packages/amesos2/src/Amesos2_Util.cpp

```diff
@@ -112,12 +109,11 @@ Amesos2::Util::to_epetra_comm(Teuchos::RCP<const Teuchos::Comm<int> > c)
     Teuchos::RCP<const Epetra_SerialComm> serialComm = rcp(new Epetra_SerialComm());
     return serialComm;
   }
-#endif	// HAVE_MPI
+#endif  // HAVE_MPI
```

Comment:
@mhoemmen were there missing `#endif`s in this file? Asking since the number of `#ifdef` removed does not match removed `#endif`s

Reply:
@ndellingwood In my changed version, the counts of `#ifdef`s and `#endif`s match.  It's possible that the code was not correct before.

Reply:
Oh goodness, thanks for clarifying!